controllers/multiclusterobservability/observatorium.go

```diff
@@ -412,7 +453,7 @@ func newStoreSpec(mco *mcov1beta2.MultiClusterObservability, scSelected string)
 	storeSpec.VolumeClaimTemplate = newVolumeClaimTemplate(
 		mco.Spec.StorageConfig.StoreStorageSize,
 		scSelected)
-	storeSpec.Shards = util.GetReplicaCount("StatefulSet")
+	storeSpec.Shards = &replicas3
```

Comment:
I do not understand why `storeSpec.Shards` value is a fixed value `replicas3 `.

Reply:
Replicas are for same data (HA). Each shard has different data, so we need different statefulset. We need to do more tests to see if we need to have more shards.